config/Zed/Oms/ComputopPayuCeeSingle01.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0"?>
+<statemachine
+    xmlns="spryker:oms-01"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="spryker:oms-01 http://static.spryker.com/oms-01.xsd">
+
+    <process name="ComputopPayuCeeSingle01" main="true">
+        <states>
+            <state name="new" reserved="true"/>
+            <state name="invalid" reserved="false"/>
+            <state name="authorize request" reserved="true"/>
+            <state name="authorized" reserved="true"/>
+            <state name="confirmed" reserved="true"/>
+            <state name="cancelled" reserved="false"/>
+            <state name="captured" reserved="true"/>
+            <state name="capture failed" reserved="false"/>
+            <state name="shipped" reserved="true"/>
+            <state name="refunded" reserved="false"/>
+            <state name="refund failed" reserved="false"/>
+            <state name="closed" reserved="false"/>
+        </states>
+
+        <events>
+            <event name="invalidation" timeout="20min" />
+            <event name="wait for payment" timeout="3days" command="Computop/Refund" />
+            <event name="confirm" onEnter="true" manual="true" command="Oms/SendOrderConfirmation"/>
+            <event name="cancel" manual="true" command="Computop/Cancel" timeout="14days"/>
+            <event name="capture" manual="true" command="Computop/Capture"/>
+            <event name="ship" manual="true"/>
+            <event name="refund" manual="true" command="Computop/Refund"/>
+            <event name="close" manual="true"/>
+        </events>
+
+        <transitions>
+            <transition condition="Computop/IsAuthorized" happy="true">
+                <source>new</source>
+                <target>authorized</target>
+            </transition>
+
+            <transition condition="Computop/IsAuthorizeRequest" happy="true">
```

Comment:
Yes if payment is for example Bank transfer it will return status 'authorize request' and we will wait for it 3 days before move to invalid. For other payment methods it will go to direct `authorized`.